### PR TITLE
dont remove amped weapons after entering titan

### PR DESCRIPTION
**Pull request by BobTheBob9**

### CODE REVIEW

**Comment by x3Karma** [comment]
Other than the misspelling looks good to ms

**Comment by GeckoEidechse** [comment]
> Other than the misspelling looks good to ms

Which to be precise? ^^

File: Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut [hunk 1]
```diff
@@ -514,8 +510,15 @@ void function PlayerUsesNukeBurncardThreaded( entity player )
 	thread TitanEjectPlayer( titan, true )
 }
 
+void function InitialisePermenantAmpedWeaponsForPlayer( entity player )
```
Comment (x3Karma) [inline Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut:513]:
```suggestion
void function InitialisePermanentAmpedWeaponsForPlayer( entity player )
```

File: Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut [hunk 2]
```diff
@@ -514,8 +510,15 @@ void function PlayerUsesNukeBurncardThreaded( entity player )
 	thread TitanEjectPlayer( titan, true )
 }
 
+void function InitialisePermenantAmpedWeaponsForPlayer( entity player )
+{
+	player.s.hasPermenantAmpedWeapons <- false
```
Comment (x3Karma) [inline Northstar.CustomServers/mod/scripts/vscripts/burnmeter/_burnmeter.gnut:515]:
```suggestion
	player.s.hasPermanentAmpedWeapons <- false
```

**Comment by RoyalBlue1** [comment]
this crashes during compile in EarnMeterMP_BoostEarned with incorrect amount of parameters. fixing the amount of parameters will revert #313

**Comment by RoyalBlue1** [comment]
Also there should probably be a death callback that resets hasPermanentAmpedWeapons to false

**Comment by RoyalBlue1** [comment]
> Would this work for the following scenario?
> 
> 1. Player gets permanent amped weapons
> 2. Player swaps out one or more of their weapons
> 3. Player enters titan
> 4. Player exits titan
> 5. Only the guns that the player had originally have amped weapons

this works as intended

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
Marking as changes requested based on @RoyalBlue1 comment: https://github.com/R2Northstar/NorthstarMods/pull/432#issuecomment-1186644671

**Comment by GeckoEidechse** [comment]
bump lol

**Comment by Zanieon** [comment]
I think this is actually safe to close because:

1. Permanent Amped Weapons is an exclusive boost to Frontier Defense
2. @RoyalBlue1 Already integrated this PR in the Frontier Defense branch even before i assumed development of it
3. Due to both above this will automatically come along together when Frontier Defense is officially merged into main

**Pull request closed without merging**